metaserver/src/main/scala/scala/meta/languageserver/Compiler.scala

```diff
@@ -118,9 +118,8 @@ class Compiler(
         s"Indexing classpath with ${sourcesClasspath.length} entries..."
       )
     }
-    ctags.Ctags.index(sourcesClasspath) { doc =>
-      documentSubscriber.onNext(doc)
-    }
+    val db = ctags.Ctags.indexDatabase(sourcesClasspath)
```

Comment:
We should rename this since it isn't actually ctags structure (although ctags generation would still be a nice separate feature).

Reply:
Not a blocker for the pr though.

Reply:
Agree, we can rename ctags to something else. Suggestion are welcome